src/fontra/client/web-components/inline-svg.js

```diff
@@ -3,11 +3,16 @@
 import { htmlToElement } from "/core/html-utils.js";
 
 export class InlineSVG extends HTMLElement {
-  constructor(src) {
+  constructor(src, options) {
```

Comment:
Instead of hacking workarounds into existing code, let's figure out the problem.

Reply:
I reverted the latest changes.